--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -49,7 +49,7 @@
 
 option(DISABLE_SSE "Disable SSE optimizations" OFF)
 
-option(BUILD_SHARED_LIBS "Build shared library" ON)
+option(BUILD_SHARED_LIBS "Build shared library" OFF)
 if(NOT BUILD_SHARED_LIBS)
   add_definitions(-DLIBDE265_STATIC_BUILD)
 endif()
--- a/libde265/de265.h
+++ b/libde265/de265.h
@@ -38,9 +38,9 @@
 
 #if defined(_MSC_VER) && !defined(LIBDE265_STATIC_BUILD)
   #ifdef LIBDE265_EXPORTS
-  #define LIBDE265_API __declspec(dllexport)
+  #define LIBDE265_API
   #else
-  #define LIBDE265_API __declspec(dllimport)
+  #define LIBDE265_API
   #endif
 #elif HAVE_VISIBILITY
   #ifdef LIBDE265_EXPORTS
